src/officehours_ui/static/css/main.css

```diff
@@ -554,6 +553,27 @@ h3.alert-heading {
 	color: #cecbc9;
 }
 
+
+.table-responsive {
+    scrollbar-width: thin;
+    scrollbar-color: rgba(0, 0, 0, .5) white;
+}
```

Comment:
Looks like this does get applied in Firefox (I see the styles with the inspector, and checking boxes). It just doesn't make it show all the time. I couldn't figure out how to do this, even adjusting the OS settings. Who knows!